cost-analyzer/values.yaml

```diff
@@ -758,12 +758,32 @@ awsstore:
   useAwsStore: false
   createServiceAccount: false
 
+federatedETL:
+  # federatedCluster indicates whether this cluster should push data to the Federated store
+  federatedCluster: false 
+  # primaryCluster indicates whether this cluster should load data from the combined section of the Federated store
+  primaryCluster: false 
+  # redirectS3Backup changes the dir of S3 backup to the Federated combined store, for using Thanos-federated data in the Federated ETL
+  # Note S3 backup should be enabled separately for this.
+  redirectS3Backup: false 
```

Comment:
Any reason why we'd ever want the federated ETl to not have an s3 backup? IE not sure why this would ever be false/true